src/app.ts

```diff
@@ -1,6 +1,8 @@
 import { autoinject } from "aurelia-framework";
 import { RouterConfiguration, Router } from "aurelia-router";
+import {PLATFORM} from 'aurelia-pal';
 import { I18N } from 'aurelia-i18n';
+import 'app.scss';
```

Comment:
I don't think it will work, i tried it before i think. but go ahead, i don't have the code currently checked out so i cannot tell for sure.